packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/invoke-model.ts

```diff
@@ -162,18 +189,25 @@ export class BedrockInvokeModel extends sfn.TaskStateBase {
       throw new Error('Output S3 object version is not supported.');
     }
 
-    this.taskPolicies = this.renderPolicyStatements();
+    //Warning to let users know about the newly introduced props
+    if (props.inputPath || props.outputPath && !isFeatureFlagEnabled) {
+      Annotations.of(scope).addWarningV2('aws-cdk-lib/aws-stepfunctions-taks',
+        'These props will set the value of inputPath/outputPath as s3 URI under input/output field in state machine JSON definition. To modify the behaviour set feature flag `@aws-cdk/aws-stepfunctions-tasks:useNewS3UriParametersForBedrockInvokeModelTask": true` and use props s3InputUri/s3OutputUri');
```

Comment:
should we change `s3InputUri/s3OutputUri` to be `input.s3InputUri/output.s3OutputUri` to make it clear